### ISSUE TITLE
508 compliance for accessibility to the blind

### ISSUE THREAD

**Issue by superjai** [marinebon/fk-esr-info#18]
Hey @bbest - so I think that increasing accessibility is something NOAA wants to do. It is a little challenging to make this content more accessible to the visually impaired, since it is mostly highcharts graphs that are being shown in the modal windows. However, I was thinking at the very least it would be good to include an alt attribute for the highcharts (it's at least something for screen readers). It looks like highcharts doesn't directly support alt, but there is a way around that. Here's the solution that has been [suggested](https://stackoverflow.com/questions/55154835/is-there-any-way-to-assign-alt-property-while-using-highcharts).

Should be pretty easy to implement here, using the chart title as the alt text. 

I am happy to go ahead with this, but wanted to get your input first.

**Comment by bbest** [comment]
Good idea to flag this. I don't think simply repeating the title adds value. If we could detect trends and report on that like we do with [ecoidx::plot_ts()](https), that would be different:

![image](https://user-images.githubusercontent.com/2837257/112659003-36358f00-8e11-11eb-8f25-7b800c001830.png)

> The index changed by less than one standard deviation of the full time series over the last 5 years (indicated by icon: →).   The mean of the last 5 years was more than one standard deviation below the mean of the full time series (indicated by icon: +).
 
But this would require processing the data (ie regular Rmd via Github Action; vs directly consuming via JS as we do now).

This concept was also originally mentioned in [Statement of Work - Google Docs](https://docs.google.com/document/d/1tG7P04XEMUPOUwu8ZOlIncWJhEbcAqM2HPIAIl-BilI/edit#heading=h.s85jzdybv84o):

Accessible Rmarkdown (508 Compliance)
Accessible R Markdown Documents
- [BrailleR](https://r-resources.massey.ac.nz/rmarkdown/)
  - [Example 1](https://cran.r-project.org/web/packages/BrailleR/vignettes/Ex1histograms.html)
  - [Example 2](https://cran.r-project.org/web/packages/BrailleR/vignettes/Ex2BasicNumerical.html)
